config/interpolate_funcs.go

```diff
@@ -95,6 +96,19 @@ func interpolationFuncFile() ast.Function {
 	}
 }
 
+// interpolationFuncBase64 implements the "base64" function to
+// encode a string in base64
+func interpolationFuncBase64() ast.Function {
```

Comment:
I don't think just adding this function is sufficient.

There's also a mapping from the names used in the configuration to these functions near the top of this file in ``func init()``, where I think you'd need to add a new entry for this.